entrypoint.sh

```diff
@@ -46,6 +46,10 @@ then
     set -x
 fi
 
+setOutput() {
+    echo "${1}=${2}" >> ${GITHUB_OUTPUT}
```

Comment:
And this one as per shellcheck the double quoting of ${GITHUB_OUTPUT}

Then is ready for merge

Reply:
Done.